src_assets/windows/misc/migration/migrate-config.bat

```diff
@@ -0,0 +1,43 @@
+@echo off
+
+rem Get sunshine root directory
+for %%I in ("%~dp0\..") do set "OLD_DIR=%%~fI"
+
+rem Create the config directory if it didn't already exist
+set "NEW_DIR=%OLD_DIR%\config"
+if not exist "%NEW_DIR%\" mkdir "%NEW_DIR%"
+
+rem Migrate all files that aren't already present in the config dir
+if exist "%OLD_DIR%\apps.json" (
+    if not exist "%NEW_DIR%\apps.json" (
+        move "%OLD_DIR%\apps.json" "%NEW_DIR%\apps.json"
+    )
+)
+if exist "%OLD_DIR%\sunshine.conf" (
+    if not exist "%NEW_DIR%\sunshine.conf" (
+        move "%OLD_DIR%\sunshine.conf" "%NEW_DIR%\sunshine.conf"
+    )
+)
+if exist "%OLD_DIR%\sunshine_state.json" (
+    if not exist "%NEW_DIR%\sunshine_state.json" (
+        move "%OLD_DIR%\sunshine_state.json" "%NEW_DIR%\sunshine_state.json"
+    )
+)
+
+rem Migrate the credentials directory
+if exist "%OLD_DIR%\credentials\" (
+    if not exist "%NEW_DIR%\credentials\" (
+        move "%OLD_DIR%\credentials" "%NEW_DIR%\"
+    )
+)
+
+rem Migrate the covers directory
+if exist "%OLD_DIR%\covers\" (
+    if not exist "%NEW_DIR%\covers\" (
+        move "%OLD_DIR%\covers" "%NEW_DIR%\"
```

Comment:
I haven't tested, but I think this would be a breaking change.

This is an example app where the cover finder feature is used.
```json
        {
            "name": "Halo",
            "output": "",
            "cmd": "",
            "image-path": ".\/covers\/igdb_740.png"
        }
```

Anything like the `sed` command we can use in batch to patch the `apps.json`? We obviously don't want to include another software just for this, but maybe we can invoke a simple powershell command from the bat script? https://www.computerhope.com/forum/index.php?topic=142440.0

I'd definitely prefer to keep the majority in batch, so if it can't be a one liner powershell command, maybe add a `patch-apps.ps` script. Wouldn't be a bad idea to also patch the program files in env/path that you previously fixed as well.

Or instead of patching apps.json we could handle it in `process.cpp`.

Or instead of over complicating this, we can just document that user's need to fix their image paths.

Reply:
The Powershell solution worked